docs/connector-development/connector-builder-ui/overview.md

```diff
@@ -1,57 +1,48 @@
 # Connector Builder Intro
 
-The connector builder UI provides an intuitive UI on top of the [low-code YAML format](https://docs.airbyte.com/connector-development/config-based/understanding-the-yaml-file/yaml-overview) and uses built connectors for syncs within the same workspace directly from within the UI. We recommend using it to iterate on your low-code connectors.
+Connector Builder is a no-code tool that’s part of the Airbyte UI. It provides an intuitive user interface on top of the [low-code YAML format](https://docs.airbyte.com/connector-development/config-based/understanding-the-yaml-file/yaml-overview) and lets you develop a connector to use in data syncs without ever needing to leave your Airbyte workspace. The connector builder offers the most straightforward method for building and maintaining connectors.
 
-:::note
-The connector builder UI is in beta, which means it’s still in active development and may include backward-incompatible changes. Share feedback and requests with us on our Slack channel or email us at feedback@airbyte.io
+We recommend that you determine whether the connector you want can be built with the connector builder before looking at the Low-Code CDK or Python CDK. 
 
-Developer updates will be announced via our #help-connector-development Slack channel. If you are using the CDK, please join to stay up to date on changes and issues.
-:::
+## Connector vs. configured source vs. connection
```

Comment:
This section feels out of place in these docs to me. Do we need to understand the difference between these things on this page? 

I think it is valuable info in general, but maybe belongs on something like a shared reference page?

Reply:
I completely agree! I think it should go into a more general "useful concepts" page and not be included here. It's in the works.